COLLABORATOR_GUIDE.md

```diff
@@ -339,13 +339,13 @@ documentation must state the deprecation status.
   * Backward-incompatible changes including complete removal of such APIs may
     occur at any time.
 
-Documentation-Only Deprecations may be handled as semver-minor or semver-major
-changes. Such deprecations have no impact on the successful operation of running
-code and therefore should not be viewed as breaking changes.
+Apply the `notable change` label to all pull requests that introduce
+Documentation-Only Deprecations. Such deprecations have no impact on code
```

Comment:
AFAIK we try to add the label to all deprecations - no matter if it's doc-only, runtime or end-of-life.

Reply:
Yes, and that's noted on line 371 currently. I'm repeating it here though because Documentation-Only deprecations are the ones most likely to be missed as notable changes. Other ones go in as semver-major so they're likely to be noticed.